metaflow/plugins/docker/fast_bakery.py

```diff
@@ -0,0 +1,172 @@
+import hashlib
+import json
+import requests
+
+from metaflow.exception import MetaflowException
+from metaflow.metaflow_config import (
+    FAST_BAKERY_TYPE,
+    FAST_BAKERY_URL,
+    FAST_BAKERY_AUTH,
+    get_pinned_conda_libs,
```

Comment:
can the environment implementation fetch the pinned dependencies and pass it to the bakery? ideally, we can ensure that this client has no dependencies on metaflow - the auth variables can be populated by docker_environment.py